_includes/v21.1/orchestration/start-cockroachdb-helm-secure.md

```diff
@@ -119,5 +113,5 @@ Secure CockroachDB deployments on Amazon EKS via Helm are [not yet supported](ht
     ~~~
 
 {{site.data.alerts.callout_success}}
-The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to a pod/node's logs to troubleshoot, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
+The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to logs for a pod or node, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
```

Comment:
```suggestion
The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to logs for a pod, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
```